CHANGELOG.md

```diff
@@ -43,6 +43,23 @@ system.io {
 }
 ```
 
+* Added a `script-epilogue` configuration option to adjust the logic that runs at the end of the scripts which wrap call executions.
+  This option is adjustable on a per-backend basis.  If unspecified, the default value is `sync`.  To turn off the default `sync` 
+  behavior on the `Local` backend:
+  
+```
+backend {
+  default = "Local"
+  providers {
+    Local {
+      actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
+      config {
+        script-epilogue = ""
```

Comment:
If it was me, I'd like to see a commented out "sync", so that I can see what is it that I'm changing it away from